lib/SecretsCrypto/sf-secrets-manager.c

```diff
@@ -274,8 +394,8 @@ static void _sf_secrets_manager_set_property(GObject *object, guint property_id,
                         g_direct_equal,
                         g_object_unref,
                         NULL);
-                priv->requests = g_hash_table_new(g_str_hash,
-                        g_str_equal);
+                priv->requests = g_hash_table_new_full(g_str_hash,
+                        g_str_equal, (GDestroyNotify)g_free, NULL);
```

Comment:
g_free() should match GDestroyNotify -> unnecessary cast?

Reply:
True.